addons/api/addon/generated/models/host-catalog.js

```diff
@@ -68,13 +69,15 @@ export default class GeneratedHostCatalogModel extends BaseModel {
 
   @attr('string', {
     for: 'plugin',
+    compositeType: 'aws',
```

Comment:
Asking for learning purposes: what is the goal of adding `compositeType` within the model?